### DISCUSSION TITLE
G1 speed

### DISCUSSION THREAD

**Discussion by Tribo-2dot0** [arkypita/LaserGRBL#1744] [General]
I'm sorry, I've asked before but I still don't understand very well... (some things in G code I understand, others I don't)
But, in the case in the image where I left G1 at 2x to make framing faster and forgot to go back to 1x...
During engraving, does the G1 only accelerate in the areas circled in red? (which is when the laser transports itself off to the next engraving point)
If so, then he would not influence the recording, correct?

![screenshot](https://user-images.githubusercontent.com/83982525/153103874-2c358a2c-6739-4989-82b9-2672187b0627.png)

**Comment by arkypita** [comment]
G1 is used for tracing movement.
For positioning movement (the area you highlight in red) G0 is used.

G1 is linear movement at programmed speed.
G0 is rapid movement at maximum speed.

Maximum speed can be configured in grbl configuration. Also acceleration matter.
Please read: https://lasergrbl.com/configuration/

**Comment by Tribo-2dot0** [comment]
Understand. My intention was to make Framing faster. I found that I can edit the button and change its F value. Thanks.


![screenshot2](https://user-images.githubusercontent.com/83982525/153220748-c965cec6-bbbb-46b0-99ba-8243cfb53007.png)
